server/lib/search.js

```diff
@@ -153,10 +141,8 @@ export const searchCollectivesInDB = async (
     } else {
       isUsingTsVector = true;
       dynamicConditions += `
-        AND (${TS_VECTOR} @@ plainto_tsquery('english', :vectorizedTerm)
-        OR ${TS_VECTOR} @@ plainto_tsquery('simple', :vectorizedTerm)
-        OR name ILIKE '%' || :term || '%'
-        OR slug ILIKE '%' || :term || '%')`;
```

Comment:
Remove these two will make the search term be exact as evidenced here; https://opencollective.slack.com/archives/G46PNRCGP/p1649215867073719. As you can see Alina is searching for `1k proj` and there's no search results whereas the exact term `1k project` gives the correct results. I think we should have these two regardless of the `searchTsVector` since the name and slug could be spelled partway (especially if it's too long). 🤔

Reply:
@SudharakaP TS vectors are meant for full-text search, they offer more options and better performances than ILIKEs. There are ways to do partial searches with them, let's not bring that back.

Reply:
Proposing fix at, https://github.com/opencollective/opencollective-api/pull/7403